core/sets.gms

```diff
@@ -766,123 +765,131 @@ sets
    ext_regi "extended regions list (includes subsets of H12 regions)"
       /
         GLO,
-        LAM_regi,OAS_regi,SSA_regi,EUR_regi,NEU_regi,MEA_regi,REF_regi,CAZ_regi,CHA_regi,IND_regi,JPN_regi,USA_regi,
-        LAM,OAS,SSA,EUR,NEU,MEA,REF,CAZ,CHA,IND,JPN,USA
+        LAM_regi,OAS_regi,SSA_regi,EUR_regi,NEU_regi,MEA_regi,REF_regi,CAZ_regi,CHA_regi,IND_regi,JPN_regi,USA_regi,EU27_regi,NEU_UKI_regi,
```

Comment:
Why are there all these region changes in sets.gms? 

Are you sure they should be committed?

Reply:
i always wonder whether it matters: when i run EU21, it changes those sets, and if i don't specifically avoid it, then it gets committed.
Just in case i'll revert to normal sets. But maybe @LaviniaBaumstark knows if that matters?

Reply:
Usually they are not committed, R scripts overwrites the GAMS sets for dynamic resolution, so it's not part of your changes per se